datacenter/dtr/2.2/guides/admin/configure/jobrunner.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Jobrunner
+description: Learn about the inner-workings of the jobrunner container in the DTR workflow.
+keywords: docker, job, runner
+---
+
+The jobrunner container is a DTR mechanism that:
+
+1. Consumes jobs from a cluster-wide jobs queue 
+2. Performs the work of the given action
+
+There is one jobrunner container per replica. 
+
+## Jobrunner Workflow
+The following diagram depicts the behavior of the jobrunner:
```

Comment:
We should mention that each jobrunner has a queue of jobs internally... also that each job is claimed after a delay of up to N seconds (maybe 3?) so that each jobrunner has a chance to compete for the job. It might also be worth mentioning that jobs that can't be picked up due to capacity limits are queued until the capacity becomes available.